.github/workflows/auto-assign-pr.yml

```diff
@@ -0,0 +1,53 @@
+name: (Privileged) Auto Assign
+
+on:
+  # Don't use pull_request or pull_request_target here.
+  # See auto-assign-pr-schedule.yaml for explanation.
+  workflow_run:
+    workflows: [ "Auto Assign PR" ]
+    types:
+    - completed
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+    - name: 'Download parameter'
+      uses: actions/github-script@v3.1.0
+      with:
+        script: |
+          var artifacts = await github.actions.listWorkflowRunArtifacts({
```

Comment:
I would use `const` here in place of all the `var`s.

`var` isn't really used in JavaScript anymore (`let` is for mutable references).

Reply:
Let me fix this later, want to first try if this is a right path